chat/src/main/java/bisq/chat/notifications/ChatNotificationService.java

```diff
@@ -362,14 +362,17 @@ private <M extends ChatMessage> ChatNotification createNotification(String id, C
         } else {
             String channelInfo;
             if (chatMessage instanceof PublicChatMessage) {
-                // For Public messages we show title: `{username} ({channel title})`
-                channelInfo = chatService.findChatChannelService(chatChannel)
+                // For Public messages we show title: `{username} ({channel domain} - {channel title})`
+                String channelTitle = chatService.findChatChannelService(chatChannel)
                         .map(service -> service.getChannelTitle(chatChannel))
                         .orElse(Res.get("data.na"));
+                channelInfo = StringUtils.createNavigationPath(
+                        chatChannel.getChatChannelDomain().getDisplayString(), channelTitle);
```

Comment:
I think a higher level Util would be better here. One which passed the channel only and the Util method build the full display string). Would need to be in the chat module as it would take ChatChannel as param.

Reply:
I agree. Somehow I understood from [your comment](https://github.com/bisq-network/bisq2/issues/1411#issuecomment-1859412380) that it would be useful in `StringUtil`. I changed this in #1561.